src/components/step-sequence/step-sequence.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './step-sequence.scss';
+import './step-sequence-orientation.scss';
+
+const baseClass = 'carbon-step-sequence';
+const classes = orientation => `${baseClass} ${baseClass}--${orientation}`;
+
+const StepSequence = ({ children, orientation }) => (
+  <ol className={ classes(orientation) }>
+    { children }
+  </ol>
+);
+
+StepSequence.propTypes = {
+  children: PropTypes.node,
+  orientation: PropTypes.oneOf([
+    'horizontal',
+    'vertical'
+  ])
+};
+
+StepSequence.defaultProps = {
+  orientation: 'horizontal'
+};
+
+export default StepSequence;
```

Comment:
Just thought, it'd be very useful if this exported both the `StepSequence` and the `StepSequenceIndicator` components so you could do this instead:

```
import { StepSequence, StepSequenceIndicator } from 'carbon-react/lib/components/step-sequence';
```

Reply:
Yep I just noticed it while using it!

That and the aria changes are the last things to do here